target/scripts/startup/setup.d/user.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+function _setup_user_uid () {
+  _log 'debug' 'Setting Custom UID if needed'
+  if "${UID_DOCKER}" != 5000; then
+    usermod -u "${UID_DOCKER}" docker
+  fi
+  if "${GID_DOCKER}" != 5000; then
+    groupmod -u "${GID_DOCKER}" docker
+  fi
```

Comment:
I prefer the long-option for additional clarity (_initially I thought `-u` meant `--update`_)

```suggestion
  if "${UID_DOCKER}" != 5000; then
    _log 'debug' "Setting 'docker' UID to ${UID_DOCKER}"
    usermod --uid "${UID_DOCKER}" docker
  fi
  if "${GID_DOCKER}" != 5000; then
    _log 'debug' "Setting 'docker' GID to ${GID_DOCKER}"
    groupmod --gid "${GID_DOCKER}" docker
  fi
```

Docs:
- https://linux.die.net/man/8/usermod
- https://linux.die.net/man/8/groupmod

`groupmod` does not have a `-u` flag:

```console
$ groupmod -u 9042 docker

groupmod: invalid option -- 'u'
Usage: groupmod [options] GROUP

Options:
  -g, --gid GID                 change the group ID to GID
  -h, --help                    display this help message and exit
  -n, --new-name NEW_GROUP      change the name to NEW_GROUP
  -o, --non-unique              allow to use a duplicate (non-unique) GID
  -p, --password PASSWORD       change the password to this (encrypted)
                                PASSWORD
  -R, --root CHROOT_DIR         directory to chroot into
  -P, --prefix PREFIX_DIR       prefix directory where are located the /etc/* files
```